package.json

```diff
@@ -48,7 +48,7 @@
     "typescript": "^5.2.2"
   },
   "peerDependencies": {
-    "@comapeo/schema": "^1.4.1"
+    "@comapeo/schema": "^1.6.0"
```

Comment:
I _think_ this change constitutes a breaking change, but could be mistaken. Open to input!

Reply:
Also part of me wonders if this should be declared as an exact direct dep instead of a non-exact peer dep. To avoid issues like https://github.com/digidem/mapeo-mock-data/issues/44

Reply:
Yeah I think that's why we haven't made it exact. I'll leave it as is and fix things as needed if a similar issue comes up again, i guess 😄